experimental/dds/tree2/src/core/tree/delta.ts

```diff
@@ -3,40 +3,35 @@
  * Licensed under the MIT License.
  */
 
-import { Brand, Opaque } from "../../util";
 import { FieldKey } from "../schema-stored";
 import { ITreeCursorSynchronous } from "./cursor";
 
 /**
- * This format describes changes that must be applied to a document tree in order to update it.
+ * This format describes changes that must be applied to a forest in order to update it.
  * Instances of this format are generated based on incoming changesets and consumed by a view layer (e.g., Forest) to
  * update itself.
  *
  * Because this format is only meant for updating document state, it does not fully represent user intentions.
- * For example, if some edit A inserts content and some subsequent edit B deletes that content, then a Delta that
- * represents the state update for these two edits would not include the insertion and deletion.
- * For the same reason, this format is also not fit to be rebased in the face of concurrent changes.
+ * For example, if some concurrent edits A and B insert content at the same location, then a Delta that represents
```

Comment:
I don't understand the example. There's also an extra "represents" at the end of the line.

Reply:
Would it be clearer if I called out tiebreaking information not being included (since that's what this example is getting at)?